### PR TITLE
Add GUI Changes on TKInter

### PR DESCRIPTION
**Pull request by AdamKBeck**

### CODE REVIEW

File: postr/twitter/twitter_graphing.csv [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Tweet,Time
+"絶対行かなきゃ✊
+@ivvy_hiroto",2018-12-10 19:14:43.016818
+"RT @AdamParkhomenko: The sad part about all of this is each time there is “news” it is not news to many Republicans who were, and are, part…",2018-12-10 19:14:43.036513
+Taking a poll please RT after voting. The MSM is fake news would you rather see a network with honest news and trut… https://t.co/iFsDEhp7bk,2018-12-10 19:14:43.037244
+@SteveDicksonQLD is spot on. Being a refugee is illegal and all they want is to come here and live lavishly on our… https://t.co/ADvJRItYOv,2018-12-10 19:14:43.038359
+すごっ！！,2018-12-10 19:14:43.060215
+Ex-Baylor frat president gets probation in rape case https://t.co/05vzEwQcgP,2018-12-10 19:14:43.076813
```
Comment (dbgrigsby) [inline postr/twitter/twitter_graphing.csv]: Send this via slack, add to gitignore

Comment (AdamKBeck) [reply postr/twitter/twitter_graphing.csv]: Sorry, will do. These are realtime tweets that my streamer picked up.

Comment (dbgrigsby) [reply postr/twitter/twitter_graphing.csv]: still there

Comment (AdamKBeck) [reply postr/twitter/twitter_graphing.csv]: I removed a different file similar to this. I have now removed this file.

**Comment by dbgrigsby** [comment]
do it

 u wont

**Pull request merged**